### DISCUSSION TITLE
Using node or npm in a path outside of C:/Windows/System32 after installing nvm

### DISCUSSION THREAD

**Discussion by tmaitland** [coreybutler/nvm-windows#997] [Support]
I installed and confirmed NVM's installation as an administrator. It exists at the C:/Windows/System32 path, but when I switched to another path where a project I want to work on lives, node or npm is not recognized. How do I get node and npm to be recognized globally via NVM? I downloaded this program to switch to different versions of node anywhere I use node or npm on my computer.

**Comment by coreybutler** [comment]
Did you actually run the install & use commands, i.e. `nvm install lts` and `nvm use lts`? Installing NVM4W only installs `nvm.exe`.

Additionally, I strongly advise _against_ using `C:\Windows\System32` as the installation root. This is a core Windows system directory that is often modified by Windows system updates. Many people use the default, `C:\Program Files`. Personally, I use `C:\nvm-windows`.
